scripts/utils.py

```diff
@@ -157,8 +168,48 @@ async def get_account(
     )
 
 
+async def contract_exists(address: int) -> bool:
+    try:
+        await GATEWAY_CLIENT.get_code(address)
+        return True
+    except ContractNotFoundError:
+        return False
+
+
+async def get_evm_account() -> AccountClient:
+    starknet_account = await get_starknet_account()
+    kakarot_contract = await Contract.from_address(KAKAROT_ADDRESS, GATEWAY_CLIENT)
+    starknet_address = (
+        await kakarot_contract.functions["compute_starknet_address"].call(
+            int(EVM_ADDRESS, 16)
+        )
+    ).contract_address
+
+    if not await contract_exists(starknet_address):
+        call = Call(
+            to_addr=int(KAKAROT_ADDRESS, 16),
+            selector=get_selector_from_name("deploy_externally_owned_account"),
+            calldata=[int(EVM_ADDRESS, 16)],
+        )
+        logger.info(f"⏳ Deploying EOA account")
+        tx_hash = (
+            await starknet_account.execute(call, max_fee=int(10e16))
+        ).transaction_hash
+        logger.info(f"⏳ Waiting for tx {get_tx_url(tx_hash)}")
+        await starknet_account.wait_for_tx(tx_hash)
+        await fund_address(starknet_address, 0.005)
```

Comment:
actually there is already a `deploy_and_fund_evm_address` that could be used here instead